src/react-components/src/__tests__/components/Accordion-test.tsx

```diff
@@ -0,0 +1,40 @@
+import { expect } from "chai";
+import * as Enzyme from "enzyme";
+import * as React from "react";
+
+import Accordion from "../../components/03-organisms/Accordion/Accordion";
+
+describe("Renders Input (closed state)", () => {
+  let container: Enzyme.ReactWrapper<any, Readonly<{}>, React.Component<{}, {}, any>>;
+  before(() => {
+    container = Enzyme.mount(<Accordion
+      id="accordion"
+      buttonOptions={{ id: "accordionBtn", content: <>Click to expand</> }}
+    > <div className="accordion-content">content content</div> </Accordion>);
```

Comment:
The `Accordion` component already renders a div with `accordion-content` css class. Now this is rendering twice for these tests. For the "open state" test, this fails `expect(container.find(".accordion-content").length).to.equal(1);`.

Reply:
Did this get updated?